operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -64,6 +64,7 @@
 import oracle.kubernetes.weblogic.domain.model.AdminService;
 import oracle.kubernetes.weblogic.domain.model.Channel;
 import oracle.kubernetes.weblogic.domain.model.Domain;
```

Comment:
Where is the Info level message we discussed in slack that informs users that no wallet file is specified, and therefore a schema creation attempt is likely about to be made?

Reply:
It's not there yet.

Reply:
Can you add it?  It can go in the java or the shell script.

Reply:
I'm OK with this pull going in if the Info message isn't in yet - it can go into another pull.

Reply:
It is there  line 677 in modelInImage.sh

  if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
    if [ -z "${OPSS_FLAGS}" ] ; then
      trace "Creating JRF Primordial Domain"
    else
      trace "Creating JRF Primordial Domain with OPSS Wallet and Password"
    fi
  fi

Reply:
That doesn't cover it. (1) Primordial is an internal term. (2) Default trace level is FINE, so it won't show up in the Operator log by default.   How about:
if [ -z "${OPSS_FLAGS}" ] ; then
trace "Creating JRF Primordial Domain"
else
trace "Creating JRF Primordial Domain with OPSS Wallet and Password"
fi

Reply:
How about:

if [ -z "${OPSS_FLAGS}" ] ; then
trace "Creating JRF Primordial Domain"
else
trace "Creating JRF Primordial Domain with OPSS Wallet and Password"
trace INFO "An OPSS wallet was not supplied for Model in the Image JRF domain 'spec.configuration.opss.walletFileSecret' attribute; therefore, it's assumed that this is the first time the OPSS RCU database is being accessed by the domain, so a schema and a wallet file will be created. Consult the Model in Image documentation for preserving the OPSS wallet file."
fi

Reply:
Let's continue the discussion on Slack if needed.

Reply:
Oops:  The INFO goes above the else!  Editing above to fix/refine.